src/Snapshotting/SnapshottingBehaviour.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace EventSauce\EventSourcing\Snapshotting;
+
+use EventSauce\EventSourcing\AggregateRoot;
+use EventSauce\EventSourcing\AggregateRootId;
+use Generator;
+
+trait SnapshottingBehaviour
+{
+    abstract public function aggregateRootVersion(): int;
+    abstract public function aggregateRootId(): AggregateRootId;
+    abstract protected function apply(object $event): void;
```

Comment:
```suggestion

    abstract protected function apply(object $event): void;
```

Reply:
What's the change here?

Reply:
white lines between the methods to stay consistent with other implementations (interfaces, etc)